ui/components/ui/box/README.mdx

```diff
@@ -177,6 +177,30 @@ import Box from '../ui/box';
 </Box>;
 ```
 
+### Color
+
+The `color` prop can be used to set the color of the content in Box Component. The `color` prop accepts [responsive props](#responsive-props) in the form of array props.
+
+<Canvas>
+  <Story id="ui-components-ui-box-box-stories-js--color" />
+</Canvas>
+
+**NOTE**: The `<Box/>` color combinations above follow our design system color rules and should cover most general UI applications. Click "Show code" to see the code example. Do not use the [deprecated colors](#deprecated-colors)
```

Comment:
We can remove this note. Deprecated colors have been removed. Can we also remove any other instances of it?

Reply:
Yes